addons/mail/static/src/new/web/chat_window/chat_window_service.js

```diff
@@ -125,7 +125,16 @@ export class ChatWindowService {
         chatWindow.thread.state = "folded";
     }
 
-    close(chatWindow, { escape = false } = {}) {
+    async close(chatWindow, { escape = false } = {}) {
+        if (this.store.isSmall && !this.store.discuss.isActive) {
```

Comment:
```suggestion
        if (this.env.isSmall && !this.store.discuss.isActive) {
```
I don't get why we have this alias in the store.

Reply:
> I don't get why we have this alias in the store.

The store is reactive, `env` is not